src/backend/distributed/deparser/deparse_statistics_stmts.c

```diff
@@ -242,7 +242,15 @@ AppendColumnNames(StringInfo buf, CreateStatsStmt *stmt)
 
 	foreach_ptr(column, stmt->exprs)
 	{
-		const char *columnName = quote_identifier(column->name);
+		if (!column->name)
+		{
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg(
+						 "only simple column references are allowed in CREATE STATISTICS")));
+		}
+
+		char *columnName = NameListToQuotedString(list_make1(makeString(column->name)));
```

Comment:
What is the reason for not using `const char *columnName = quote_identifier(column->name);` and switch to this complex expression?